build-system/SERVING.md

```diff
@@ -18,17 +18,18 @@ limitations under the License.
 
 ## Requirements
 - git
-- node + npm
+- node 6+
 - gulp (installed globally)
 - java 8
+- yarn 1.0.2+ (see https://yarnpkg.com/)
 
 ## Steps
 ```bash
 git clone https://github.com/ampproject/amphtml.git
 cd amphtml
 # Checkout a tag
 git checkout 123456789
-npm install
+yarn
```

Comment:
Does this need to be `yarn install`? Or is `yarn` sufficient if there's a `yarn.lock`?

Reply:
"yarn" == "yarn install", regardless of yarn.lock or not

https://yarnpkg.com/lang/en/docs/cli/install/

Reply:
Cool, thanks for checking!